conflate/conflate.py

```diff
@@ -78,11 +79,13 @@ def __init__(self, ptype, pid, version, lat, lon, tags=None, categories=None):
         self.members = None
         self.action = None
         self.categories = categories or set()
+        self.remarks = None
 
     def copy(self):
         """Returns a copy of this object, except for members field."""
         c = OSMPoint(self.osm_type, self.osm_id, self.version, self.lat, self.lon, self.tags.copy())
         c.action = self.action
+        c.remarks = self.remarks
```

Comment:
Looks like I forgot to copy the `categories` set. Could you do that please?

Reply:
Of course. Done.

Reply:
Thanks, though you need to use `self.categories.copy()`, otherwise you're just copying a pointer to the same set.

Reply:
My bad, corrected. Thanks for your time and the effort on this project!